README.md

```diff
@@ -106,32 +106,35 @@ bash stop-containers-and-clean-up-traces-if-existent.sh
 
 
 
-### Deployment for Production
+### Regular Deployment
 
-This approach is meant for developers and people who would like to deploy a production-ready CoatRack instance with a clean, empty database and without an Example-Gateway. Just go to the directory `docker-compose-deployment` and execute:
+This approach is meant for developers and people who would like to deploy a CoatRack instance with a clean, empty database and without an Example-Gateway. Just go to the directory `docker-compose-deployment` and execute:
```

Comment:
```suggestion
This approach is meant for those who would like to deploy a new CoatRack instance with a clean, empty database. Just go to the directory `docker-compose-deployment` and execute:
```

Reply:
As discussed, I just removed this section from the README for now and saved your suggestions for later when all features are working and this section could be re-added.